### PR TITLE
International factors data clean up

### PR DESCRIPTION
**Pull request by hcarslaw**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-3490

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/tasklistresponse/international-factors/InternationalFactorsMapper.ts [hunk 1]
```diff
@@ -1,116 +1,72 @@
...
-        parentsAnyOneLiveOutsideEnWlDetails: req.session.userCase.iFactorsParentsProvideDetails,
-
-        anotherPersonOrderOutsideEnWl: req.session.userCase.jurisdiction,
-        anotherPersonOrderOutsideEnWlDetails: req.session.userCase.iFactorsJurisdictionProvideDetails,
-
-        anotherCountryAskedInformation: req.session.userCase.request,
-        anotherCountryAskedInformationDetaails: req.session.userCase.iFactorsRequestProvideDetails,
-      },
-    };
+export const prepareInternationalFactorsRequest = (req: CaseWithId): CitizenInternationalElements => {
```
Comment (vivek-sekhar) [inline src/main/steps/tasklistresponse/international-factors/InternationalFactorsMapper.ts:4]: The argument is caseData and not req.

File: src/main/steps/tasklistresponse/international-factors/InternationalFactorsMapper.ts [hunk 2]
```diff
@@ -1,116 +1,72 @@
...
+    delete internationalElementsRequest.anotherPersonOrderOutsideEnWlDetails;
   }
-  if (respondent?.value?.response?.citizenInternationalElements?.anotherCountryAskedInformation === YesOrNo.YES) {
-    Object.assign(tempValue, {
-      request: YesOrNo.YES,
-      iFactorsRequestProvideDetails:
-        respondent?.value?.response?.citizenInternationalElements?.anotherCountryAskedInformationDetaails,
-    });
+
+  if (request === YesOrNo.NO) {
```
Comment (vivek-sekhar) [inline src/main/steps/tasklistresponse/international-factors/InternationalFactorsMapper.ts:40]: what does request mean? What data exactly does the variable hold?

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/tasklistresponse/international-factors/InternationalFactorsMapper.ts (hunk 2 above)
Comment (hcarslaw) [reply src/main/steps/tasklistresponse/international-factors/InternationalFactorsMapper.ts:40]: request is the name of the radio button for the international-factors/request page and it holds the value of that radio button

File: src/main/steps/tasklistresponse/international-factors/InternationalFactorsMapper.ts (hunk 1 above)
Comment (hcarslaw) [reply src/main/steps/tasklistresponse/international-factors/InternationalFactorsMapper.ts]: Changed req to caseData

**Review by vivek-sekhar — APPROVED** [review]

**Pull request merged**